src/components/WordList.js

```diff
@@ -1,15 +1,11 @@
 import React, { Component } from 'react'
 
 class WordList extends Component {
-  constructor(props) {
-    super(props)
-  }
-
   render() {
-    const wordListItems = this.props["wordList"].map((word) => <li>{word}</li>)
+    const wordListItems = this.props.wordList.map((word) => <li>{word}</li>)
```

Comment:
I'd just do this inline on line 8 instead of making a variable for it.

Reply:
Also need a key.